src/main/Extensions/Base64Conversion/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Base64Conversion";
```

Comment:
Please export the `Base64ConversionModule` only here, as it's the only thing that is needed outside of this folder.